hoomd/hpmc/compute.py

```diff
@@ -117,11 +117,34 @@ class SDF(Compute):
             bin :math:`[\mathrm{length}]`.
         dx (float): Bin width :math:`[\mathrm{length}]`.
 
-    `SDF` computes a distribution function of parameter :math:`x`. For each pair
-    of particles, it scales the particle separation vector by :math:`1+x` and
-    records the smallest :math:`x` that would cause the particles to touch in
-    the histogram :math:`s(x)`. The histogram is discrete and :math:`s(x_i) =
-    s[i]` where :math:`x_i = i \cdot dx + dx/2`.
+    `SDF` computes the proability distribution of particles overlapping
```

Comment:
There is no need to review `hpmc/compute.py` on this pull request. I have already rewritten this documentation for the next pull request.